src/components/app/app.js

```diff
@@ -41,6 +46,11 @@ export default class App extends Component {
         notes: notes,
       };
     });
+
+    fetch("https://jsonplaceholder.typicode.com/posts", {
+      method: "POST",
+      body: JSON.stringify(newNote),
+    });
```

Comment:
let's add a log for result of the query, so you can see that it was successful 
either then or await result 
```
  .then((response) => response.json())
  .then((json) => console.log(json));
```